requirements.txt

```diff
@@ -105,6 +105,9 @@ https://github.com/rkabadi/pyedimax/archive/master.zip
 # RPI-GPIO platform (*.rpi_gpio)
 RPi.GPIO >=0.5.11
 
+# Adafruit temperature/humidity sensor
+git+git://github.com/mala-zaba/Adafruit_Python_DHT
```

Comment:
It is breaking CI because it does not install if it is not on a Rasperry Pi or Beagle board. If you could comment it out, people can still see it is needed but it will no longer be auto installed.